crypto/provider_core.c

```diff
@@ -49,6 +49,16 @@ struct ossl_provider_st {
     STACK_OF(INFOPAIR) *parameters;
     OPENSSL_CTX *libctx; /* The library context this instance is in */
     struct provider_store_st *store; /* The store this instance belongs to */
+#ifndef FIPS_MODE
+    /*
+     * In the FIPS module inner provider, this isn't needed, since the
```

Comment:
where is this "inner" and "outer" provider terminology explained?

Reply:
Not really anywhere.  `providers/fips/fipsprov.c` does mention "this inner instance" once, that's what I'm hooking into.

You're right that it should be better documented.  Feel free to raise an issue for that.  *This* PR isn't the right place.

Reply:
oh, sigh.  you're writing comments that contain magic undefined terms.  if not here, when and where?

Reply:
I know we were each pushing the other to either open and issue or fix this PR.  I opened #9182. I hope it gets fixed somehow somewhere by someone sometime.

https://www.bartleby.com/73/2019.html